### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by DmitryPR**
General improvements to the documentation and minor typo fixes

### CODE REVIEW

File: docs/Screensharing.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # Screensharing
 
+ScreenSharing enables displaying streaming screen content shared by the Visitor
+during an Engagement.
+
 See [explanation video][0]
 
+## Setup
+
+- An iOS device running iOS 11.0+
```
Comment (urmastalimaa) [inline docs/Screensharing.md]: What happens on an earlier version?

File: docs/Screensharing.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+- No active Screencast or Airplay enabled on the device
+- Site configured to accept Visitor's screen
```
Comment (urmastalimaa) [inline docs/Screensharing.md]: What does this mean for the integrator?

File: docs/Screensharing.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+
+## Annotations
+
+When there is an active Engagement the Operator can drawn simple shapes on the screen which will be translated
```
Comment (urmastalimaa) [inline docs/Screensharing.md]: Wording is a bit clumsy, what about

`When Operator is engaged with a Visitor and the Visitor has shared their screen, Operator can draw on the _screensharing_ view and the drawing will be drawn on the Visitor screen overlaying the application.`

Also, does this drawing work for arbitrary applications or only the one using our SDK?

File: docs/FAQ.md [hunk 4]
```diff
@@ -33,3 +34,7 @@ We are working on experimental static library that can also [speed up the launch
 Our client library minimum requirement is Xcode 10.0 and Swift 4.2 
 
 One possible way of using the client library is [updating the Swift version to 4.2](https://useyourloaf.com/blog/upgrading-to-swift-4.2/)
+
+## Screensharing is not showing the screen
+
+There might be rare cases when the iOS system stops recording the screen due to their internal failure. The possible solution for this is to restart the device and try screen sharing once again.
```
[the remark this answers is not in the document]
Comment (ostap0207) [reply docs/FAQ.md]: Indeed, is there a way to detect those cases? How often is rare?

File: docs/Screensharing.md (hunk 3 above)
Comment (DmitryPR) [reply docs/Screensharing.md]: Only our sdk

File: docs/FAQ.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (DmitryPR) [reply docs/FAQ.md]: True, let's keep this internal

[the remark this answers is not in the document]
Comment (DmitryPR) [reply docs/FAQ.md]: @ostap0207 unfortunately not

File: docs/Screensharing.md (hunk 3 above)
Comment (urmastalimaa) [reply docs/Screensharing.md]: Can you specify that in the documentation?

**Comment by DmitryPR** [comment]
!squash

**Comment by DmitryPR** [comment]
!merge

**Pull request merged**